### ISSUE TITLE
"Delete" keyboard shortcut does not work if anything inside the mail was clicked or marked

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#6804]
### Steps to reproduce

1. Open any email
2. Mark some text or click a link
3. Press "Delete" on the keyboard

### Expected behavior

The mail gets deleted.

### Actual behavior

Nothing happens.

### Mail app version

Latest as deployed on cloud.nextcloud.com

**Comment by ChristophWurst** [comment]
> the message

Let me guess, HTML message?

**Comment by jancborchardt** [comment]
@ChristophWurst yep, only happens for HTML messages, not for normal ones – just tested on cloud.nc.
Is it possible to still catch the shortcut even if the focus is inside the HTML message?

**Comment by ChristophWurst** [comment]
> Is it possible to still catch the shortcut even if the focus is inside the HTML message?

It's an iframe and we sandbox it because we don't trust the HTML. We might be able to use postMessage to communicate between iframe and the main window, and tunnel events this way.
This is too big of a task for a drive-by fix -> planning

**Comment by gs** [comment]
Draft PR #13725 fixes this. vue-shortkey's `prevent` list contained `div`, and Nextcloud focuses plain divs (`#app-content`, the envelope list), so shortcuts were suppressed almost everywhere. The list now excludes only real editable elements (`input`, `textarea`, `contenteditable`).
